plugins/ltac/tauto.ml

```diff
@@ -242,7 +241,7 @@ let tauto_uniform_unit_flags = {
 }
 
 (* This is the compatibility mode (not used) *)
-let tauto_legacy_flags = {
+let _tauto_legacy_flags = {
```

Comment:
@herbelin I see, thanks for the explanation! If it is an 8.4 setting, maybe 8.7 is the right time to remove it, I guess.